CHANGELOG.md

```diff
@@ -9,9 +9,12 @@
 
 ### Fixed
```

Comment:
Also fixed seccomp edge case: https://github.com/firecracker-microvm/firecracker/issues/1206 if it's worth mentioning.

Reply:
Added.